test/average-test.js

```diff
@@ -0,0 +1,24 @@
+const assert = require('chai').assert
+const {arrayTotal, arrayMean, arrayMode} = require('../lib/average')
+
+describe('return handrolled array methods', function() {
+  it('give total', function() {
```

Comment:
i'd be more specific here within your `it`'s. what is this test actually testing? probably something like "returns the summed value of the elements"